spec/requests/apidocs_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe 'Apidocs', type: :request do
```

Comment:
Will these tests fail if the API docs are disabled?

Reply:
We can only remove apidocs action then we should remove tests.

Reply:
We should have it set from configuration so it is on in lower environments but can be turned off in prod.